cindy/index.js

```diff
@@ -0,0 +1,37 @@
+//Basic Arithmetic Problems
+
+//Q1. In one night, a movie theater sells tickets for 6450 dollars. Each ticket costs 15 dollars. How many tickets were sold?
+let dollars = 6450;
+let ticket = 15;
+let result = dollars / ticket;
+console.log(result + " tickets were sold");
+
+//Q2. Sylvia's income is 500 dollars per week. How much does Sylvia makes every year?
+let incomeWeek = 500;
+let weeks = 32;
```

Comment:
There is 52 weeks per year and not 32!!! So you have to multiply 500 to 52 instead!

Reply:
omg that was my mistake :c fixed thanks for letting me know !

Reply:
You're welcome:)